sumOfOther.js

```diff
@@ -0,0 +1,12 @@
+function sumOfOther (arr) {
+    let arr_2=[];
```

Comment:
@Greg1996 Use spaces around the operators' symbols. Rename this variable. Probably "result" will be better?